Source/WebCore/platform/graphics/avfoundation/AudioVideoRendererAVFObjC.mm

```diff
@@ -1473,6 +1463,18 @@ @interface AVSampleBufferAudioRenderer (WebCoreSampleBufferKeySession) <AVConten
     m_enabledVideoTrackId.reset();
 }
 
+void AudioVideoRendererAVFObjC::removeRendererFromSynchronizerIfNeeded(id renderer)
+{
+    bool isAVSBDL = dynamic_objc_cast<AVSampleBufferDisplayLayer>(renderer);
```

Comment:
```suggestion
    bool isAVSBDL = is_objc<AVSampleBufferDisplayLayer>(renderer);
```

Reply:
> 

didn't know of that methods, there's a few places `dynamic _objc_cast` is used for `is_objc`